docs/how-to-guides/scaling-feast.md

```diff
@@ -0,0 +1,27 @@
+# Scaling Feast
+
+## Overview
+
+Feast is designed to be easy to use and understand out of the box, with as few infrastructure dependencies as possible. However, there are components used by default that may not scale well. 
+However, since Feast is desgined to be modular, it's possible to swap such components with more performant components, at the cost of Feast depending on additional infrastructure.
```

Comment:
typo: designed

also nit: just "Since Feast is designed to be modular", to avoid repeating "However"